.travis.yml

```diff
@@ -1,33 +1,63 @@
 sudo: false
 language: python
-python: 3.5
-cache:
-  directories:
-  - "$HOME/.cache/pip"
-install:
-- pip install tox
-env:
-  - TOXENV=py27-django-18
-  - TOXENV=py27-django-19
-  - TOXENV=py27-django-110
-  - TOXENV=py27-django-111
-  - TOXENV=py35-django-18
-  - TOXENV=py35-django-19
-  - TOXENV=py35-django-110
-  - TOXENV=py35-django-111
-  - TOXENV=py35-django-master
-  - TOXENV=pypy-django-18
-  - TOXENV=pypy-django-19
-  - TOXENV=pypy-django-110
-  - TOXENV=pypy-django-111
+cache: pip
+
 matrix:
   include:
+    - python: 2.7
```

Comment:
We should switch to [tox-travis](http://tox-travis.readthedocs.io/en/stable/) instead.